community/elections/2017/vote_for_justinsb.md

```diff
@@ -0,0 +1,70 @@
+## Vote for justinsb!
+
+## A little about me:
+
+I have been involved with Kubernetes since before 1.0, primarily on AWS support
+(I am a lead of sig-aws), but also contributed the original multi-zone &
+NodePort support.  I also started the kops project, to produce an open-source
```

Comment:
There is an extra white space at the beginning of every sentence in the raw file, is it intentional ? Anyway, it's ok when view it as an markdown file.

Reply:
The two spaces after a full-stop?  It's an age-old argument, and my habit is two spaces.  Thankfully markdown doesn't care!

But I did a little search and replace anyway :-)

Reply:
Cool!